content/md/permissions-settings/access-rights-on-products.md

```diff
@@ -134,6 +134,10 @@ If a user **has rights to only edit product information** for the products that
 *   Create a product [draft](proposals-workflow.html#contribute-to-a-working-copy) that will need to be approved by the product manager
 *   Generate/send a proposal that will be processed by the product manager
 
+:::info
+To be able to access the `Categories` tab of the products that are in this given category, the user must be granted with the `Consult the categories of a product` role permission (`System` > `Roles` > `Products` > `Consult the categories of a product`). The user will be able to consult the categories but not edit them. Without this permission, the `Categories` tab won't appear.
```

Comment:
why is this under "The edition rights" section? This is the same for "The view right"

Reply:
oui en effet ! bien vu ! merci